airbyte-integrations/connectors/source-salesforce/unit_tests/unit_test.py

```diff
@@ -99,7 +100,7 @@ def stream_api_v2(stream_config):
 
 
 def _generate_stream(stream_name, stream_config, stream_api, state=None):
-    return SourceSalesforce.generate_streams(stream_config, [stream_name], stream_api, state=state)[0]
+    return SourceSalesforce.generate_streams(stream_config, {stream_name: None}, stream_api, state=state)[0]
```

Comment:
@antixar Is there any reason that the stream_name is being set to `None` here instead of the `stream_name` parameter?